docs/2.9.0/docs/canton/usermanual/performance.rst

```diff
@@ -171,7 +171,7 @@ Typically `X` only contains the receiver of the transfer, but not the contract a
 
 Then, we observe the following storage usage:
 
-- Two encrypted envelopes with payload `Y` each, one symmetric key per
+- Two encrypted envelopes with payload `Y` each, two symmetric keys per
```

Comment:
this is slightly outdated, right? 1) view seed per view, 2) symmetric key per informee group